scripts/monitoring/cron-send-security-updates-count.py

```diff
@@ -0,0 +1,218 @@
+        #       { 'advisory': 'RHSA-2000:4321', 'type': 'Moderate/Sec.', 'package': 'somepkt-3:0.19.3-14.el7.x86_64' } ]
+
+        #for errline in filter(lambda x: x and 'Errno' in x, stdout.split('\n')):
+        for errline in [x for x in stdout.split('\n') if x and 'Errno' in x]:
+            if self.verbose:
+                print('Error found:', errline)
+            logging.error('Error found:' + errline)
+        for update in self.sec_updates:
+            msg = 'Security update: {0} {1} {2}'.format(update['advisory'],
+                                                        update['type'],
+                                                        update['package'])
+            if update['advisory'] in self.sec_advisory_whitelist:
+                if self.verbose:
+                    print('Whitelisted {0}'.format(msg))
+                logging.info('Whitelisted %s', msg)
+            else:
+                if self.verbose:
+                    print(msg)
+                logging.info(msg)
+
+    def get_updates_list(self):
+        ''' Return a list of pending security updates '''
+        #return filter(lambda x: x['advisory'] not in self.sec_advisory_whitelist, self.sec_updates)
+        return [x for x in self.sec_updates if x['advisory'] not in self.sec_advisory_whitelist]
+
+    def get_updates_whitelisted_list(self):
+        ''' Return a list of whitelisted pending security updates '''
+        #return filter(lambda x: x['advisory'] in self.sec_advisory_whitelist, self.sec_updates)
+        return [x for x in self.sec_updates if x['advisory'] in self.sec_advisory_whitelist]
+
+def setup():
+    '''
+    Make a root dir where yum can be run. Bind mount parts that
+    can be used as-is, copy and edit the rest.
+    '''
+    # we need to edit some config files. Make a copy of all config files.
+    shutil.rmtree(ROOT + '/etc/yum', ignore_errors=True)
+    shutil.rmtree(ROOT + '/etc/yum.repos.d', ignore_errors=True)
+    shutil.copytree('/host/etc/yum', ROOT + '/etc/yum')
+    shutil.copytree('/host/etc/yum.repos.d', ROOT + '/etc/yum.repos.d')
+    shutil.copyfile('/host/etc/yum.conf', ROOT + '/etc/yum.conf')
+
+    # edit yum.conf
+    config = ConfigObj(ROOT + '/etc/yum.conf')
+    config['main']['cachedir'] = ROOT + '/var/cache/yum/$basearch/$releasever'
+    config['main']['reposdir'] = ROOT + '/etc/yum.repos.d'
+    config['main']['installroot'] = ROOT
+    config['main']['logfile'] = '/var/log/hostpkg.yum.log'
+    config.write()
+
+    # edit the repos using fileinput since ConfigParser doesn't support the multi-line format used by yum's baseurl tag
+    for line in fileinput.input(glob.glob(ROOT + '/etc/yum.repos.d/*.repo'), inplace=True):
+        #NOTE: WARNING: do NOT print anything here. STDOUT is being captured
+        #               and written to the repo config file via in-place editing
+        myre = r'^(?P<field>(?:gpgkey|sslcacert|sslclientkey|sslclientcert)\s*=\s*)(?P<path>.+?)(?P<tail>\s*(?:#.*)?)$'
```

Comment:
maybe a description around what your intentions are.  What are you going to do with the results?